include/common.h

```diff
@@ -11,5 +11,6 @@
 #include "si.h"
 #include "messages.h"
 #include "ld_addrs.h"
+#include "stdarg.h"
```

Comment:
I don't think this should be in common; even in typical C projects its system-included as `#include <stdarg.h>`